bool MUTE_RELEASE
---
bool success
